upload-part-file.sh

```diff
@@ -10,8 +10,6 @@ export AWS_REGION=${AWS_REGION}
 pip install openpyxl
 pip install simplejson
 
-#chmod +x ./tools/scripts/retrieve-fms-file.py
```

Comment:
@saquino0827 could you leave it there, so we can use it when we can hit auth endpoints from codebuild for automation?

Reply:
K i'm adding it back commented but it will need to upload the participation file into s3 where the rest of the script looks to download and reformat the participation file.